### PR TITLE
chore: drop duplicate get_summary() call

### PR DESCRIPTION
**Pull request by josegonzalez**

### CODE REVIEW

**Comment by josegonzalez** [comment]
CLA signed, but no, I will not provide my address for the CLA.

**Comment by jezdez** [comment]
@anaconda-issue-bot check

**Comment by jezdez** [comment]
Hi @josegonzalez, blast from the past! I think this needs only a slight update to the news files, do you have the spare cycles to do that?

**Comment by kenodegard** [comment]
pre-commit.ci autofix

**Review by kenodegard — APPROVED** [review]

File: news/3998-drop-duplicate-get_summary-call [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+* <news item>
+
+### Docs
+
+* <news item>
+
+### Other
+
+* Drop duplicate `get_summary` call. (#3998)
```
Comment (dholth) [inline news/3998-drop-duplicate-get_summary-call:19]:
```suggestion
* Drop duplicate `get_summary` call in skeletons/pypi. (#3998)
```

**Review by kenodegard — APPROVED** [review]

**Comment by kenodegard** [comment]
Failures are caused by recent release of OpenSSL 3.0.8 to defaults, waiting on https://github.com/conda/conda-build/pull/4912 to proceed

**Review by jezdez — APPROVED** [review]

**Comment by kenodegard** [comment]
ugh the remaining failures aren't flaky after all and are rather due to https://github.com/conda/conda-build/issues/4895...

**Pull request merged**